corehq/apps/fixtures/dbaccessors.py

```diff
@@ -31,8 +31,8 @@ def get_fixture_data_types(domain, bypass_cache=False):
     ))
 
 
-def get_fixture_data_type_by_tag(domain, tag):
-    data_types = get_fixture_data_types(domain)
+def get_fixture_data_type_by_tag(domain, tag, **kw):
```

Comment:
Curious to know if there is a specific reason to use `**kw` instead of adding `bypass_cache` in the definition.

Reply:
Doesn't matter much in this case, but `**kw` allows additional arguments to be added to the method that the keyword arguments are being passed to (`get_fixture_data_types`) without having to update this function.